plugins/SnapOnMove/src/main/java/org/micromanager/plugins/snaponmove/MainController.java

```diff
@@ -440,6 +451,19 @@ public void onShutdownCommencing(ShutdownCommencingEvent e) {
       }
    }
 
+   @Subscribe
+   public void onAcquisitionStarted(AcquisitionStartedEvent e) {
+      pausedForAcquisition_ = true;
+   }
+
+   @Subscribe
+   public void onAcquisitionEnded(AcquisitionEndedEvent e) {
+      synchronized (pauseLock_) {
+         pauseLock_.notify();
+      }
+      pausedForAcquisition_ = false;
```

Comment:
The setting to `false` should be inside the `synchronized` block and _before_ the call to `notify()`.